front/src/applications/operationalStudies/components/SimulationResults/ChartHelpers/drawText.ts

```diff
@@ -2,7 +2,7 @@ import { Chart } from 'reducers/osrdsimulation/types';
 
 const drawText = (
   chart: Chart,
-  direction: boolean,
+  direction: boolean | undefined,
```

Comment:
```suggestion
  direction?: boolean,
```

Reply:
This suggestion involves to change the signature of the function because we have to move direction at the end of arguments list. So we'll have to change all the places where the function is called. Not sure it's worth it